packages/components/src/components/base/ColorIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent, ReactNode } from "react";
+
+interface Props {
+    cls?: string;
+    value: string;
+    asSquare?: boolean;
+    label?: string;
+}
+
+export class ColorIcon extends PureComponent<Props> {
+    private defaultProps = {
+        cls: 'color-picker__chip',
+    };
+
+    render(): ReactNode {
+        const { cls, value, asSquare, label } = this.props;
+        const isWhite = value?.toLowerCase() === '#ffffff';
+        const border = isWhite ? 'solid 1px #000000' : undefined;
+
+        let icon;
+        if (value) {
+            icon = <i className="fa fa-circle" style={{ color: value }} />;
+        } else if (isWhite) {
+            icon = <i className="fa fa-circle-thin" />;
+        }
```

Comment:
Yes, this logic was wrong. I have done away with the use of fa-circle-thin since I realized that our apps have gray background. So maybe that means that we should be using fa-circle-thin for the gray value...argh.

Reply:
Actually, it turns out that the body background color gray we use in the apps doesn't match one of the grays in the color picker options.

Reply:
But I think you can set the hex value directly as well.

Reply:
You can? I haven't seen that ability, do you know the steps for doing that?

Reply:
I think I double-clicked on the hex value itself.  Will have to switch back to that branch to be sure.

Reply:
Yes, double-click on the hex or RGB values.  (I was trying to find a  way to clear the color choice.)